cli/ts/utils.ts

```diff
@@ -142,6 +142,21 @@ const isPathExist = (paths: Array<string>): [boolean, string] => {
     return [true, null]
 }
 
+const CompareOnchainValue = (
```

Comment:
```suggestion
const compareOnchainValue = (
```
General convention - functions should always be camelCase, ya?

Noting this commit suggestion would require updating all uses of this function...

Reply:
Edit - also capitalized the "C". See comment below for context

Reply:
> General convention - functions should always be camelCase, ya?
> 
> Noting this commit suggestion would require updating all uses of this function...

Good catch! Reflected in [974ab46](https://github.com/privacy-scaling-explorations/maci/pull/585/commits/974ab462a55ad39f0df76ce16972c53446480589)